packages/env/README.md

```diff
@@ -349,6 +349,8 @@ Options:
   --help     Show help                                                 [boolean]
   --version  Show version number                                       [boolean]
   --debug    Enable debug output.                     [boolean] [default: false]
+  --env-cwd  The working directory the command is ran from.
```

Comment:
Let's mention that this the cwd in the docker container (or something about it being internal)